src/mixins/name-request-mixin.ts

```diff
@@ -12,22 +12,23 @@ export default class NameRequestMixin extends Vue {
    * Returns True if the Name Request data is valid.
    * @param nr the name request response payload
    */
-  isNrValid (nr: any): boolean {
-    return Boolean(nr &&
+  isNrValid (nr: NameRequestIF): boolean {
+    return Boolean(
+      nr &&
       nr.state &&
       nr.expirationDate &&
       !!this.getNrApprovedName(nr) &&
-      // workaround for old or new property name
-      (nr.nrNum || nr.nrNumber) &&
```

Comment:
Workaround no longer needed. (All past NRs are long expired. All current NRs store NR number in `nrNum`.)